<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DoctorEditView</class>
 <widget class="QWidget" name="DoctorEditView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <spacer name="verticalSpacer">
       <property name="styleSheet">
        <string notr="true">/* 整体窗口背景颜色 */
QWidget#DoctorEditView {
    background-color: #ffffff; /* 白色背景 */
    font-family: &quot;Microsoft YaHei UI&quot;, sans-serif; /* 使用微软雅黑字体 */
    font-size: 12pt;
    border-radius: 5px; /* 圆角 */
    padding: 20px; /* 内边距 */
}

/* 标签样式 */
QLabel {
    font-size: 12pt;
    font-weight: bold;
    color: #007BFF; /* 蓝色字体 */
}

/* 输入框样式 */
QLineEdit {
    border: 1px solid #ccc; /* 边框颜色 */
    border-radius: 4px; /* 圆角 */
    padding: 5px;
    background-color: white;
    font-size: 14px;
}

/* 聚焦时 QLineEdit 样式 */
QLineEdit:focus {
    border-color: #007BFF; /* 聚焦时的边框颜色 */
    box-shadow: 0 0 5px rgba(0, 123, 255, 0.5); /* 聚焦时的阴影效果 */
}

/* 按钮样式 */
QPushButton {
    background-color: #007BFF; /* 蓝色背景 */
    color: white;
    border: none;
    border-radius: 5px; /* 圆角 */
    padding: 10px 20px; /* 按钮内边距 */
    font-size: 12px;
    cursor: pointer; /* 鼠标悬停时变成手型 */
}

/* 按钮悬停时的样式 */
QPushButton:hover {
    background-color: #0056b3; /* 悬停时的背景颜色 */
}

/* 保存按钮的特定样式 */
QPushButton#pushButton {
    background-color: #28a745; /* 绿色背景 */
}

QPushButton#pushButton:hover {
    background-color: #218838; /* 悬停时的绿色背景 */
}

/* 取消按钮的特定样式 */
QPushButton#pushButton_2 {
    background-color: #f44336; /* 红色背景 */
}

QPushButton#pushButton_2:hover {
    background-color: #d32f2f; /* 悬停时的红色背景 */
}

/* 布局间距 */
QGridLayout {
    margin: 20px;
    spacing: 10px; /* 控件之间的间距 */
}

/* 输入框和标签之间的间距 */
QGridLayout QLabel, QGridLayout QLineEdit {
    margin-bottom: 10px;
}

/* 添加间隔的样式 */
QSpacerItem {
    background-color: transparent;
}
</string>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>178</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0">
      <layout class="QGridLayout" name="gridLayout_2"/>
     </item>
    </layout>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="pushButton">
     <property name="styleSheet">
      <string notr="true">/* 整体窗口背景颜色 */
QWidget#DoctorEditView {
    background-color: #ffffff; /* 白色背景 */
    font-family: &quot;Microsoft YaHei UI&quot;, sans-serif; /* 使用微软雅黑字体 */
    font-size: 12pt;
    border-radius: 5px; /* 圆角 */
    padding: 20px; /* 内边距 */
}

/* 标签样式 */
QLabel {
    font-size: 12pt;
    font-weight: bold;
    color: #007BFF; /* 蓝色字体 */
}

/* 输入框样式 */
QLineEdit {
    border: 1px solid #ccc; /* 边框颜色 */
    border-radius: 4px; /* 圆角 */
    padding: 5px;
    background-color: white;
    font-size: 14px;
}

/* 聚焦时 QLineEdit 样式 */
QLineEdit:focus {
    border-color: #007BFF; /* 聚焦时的边框颜色 */
    box-shadow: 0 0 5px rgba(0, 123, 255, 0.5); /* 聚焦时的阴影效果 */
}

/* 按钮样式 */
QPushButton {
    background-color: #007BFF; /* 蓝色背景 */
    color: white;
    border: none;
    border-radius: 5px; /* 圆角 */
    padding: 10px 20px; /* 按钮内边距 */
    font-size: 12px;
    cursor: pointer; /* 鼠标悬停时变成手型 */
}

/* 按钮悬停时的样式 */
QPushButton:hover {
    background-color: #0056b3; /* 悬停时的背景颜色 */
}

/* 保存按钮的特定样式 */
QPushButton#pushButton {
    background-color: #28a745; /* 绿色背景 */
}

QPushButton#pushButton:hover {
    background-color: #218838; /* 悬停时的绿色背景 */
}

/* 取消按钮的特定样式 */
QPushButton#pushButton_2 {
    background-color: #f44336; /* 红色背景 */
}

QPushButton#pushButton_2:hover {
    background-color: #d32f2f; /* 悬停时的红色背景 */
}

/* 布局间距 */
QGridLayout {
    margin: 20px;
    spacing: 10px; /* 控件之间的间距 */
}

/* 输入框和标签之间的间距 */
QGridLayout QLabel, QGridLayout QLineEdit {
    margin-bottom: 10px;
}

/* 添加间隔的样式 */
QSpacerItem {
    background-color: transparent;
}
</string>
     </property>
     <property name="text">
      <string>保存</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="pushButton_2">
     <property name="styleSheet">
      <string notr="true">/* 整体窗口背景颜色 */
QWidget#DoctorEditView {
    background-color: #ffffff; /* 白色背景 */
    font-family: &quot;Microsoft YaHei UI&quot;, sans-serif; /* 使用微软雅黑字体 */
    font-size: 12pt;
    border-radius: 5px; /* 圆角 */
    padding: 20px; /* 内边距 */
}

/* 标签样式 */
QLabel {
    font-size: 12pt;
    font-weight: bold;
    color: #007BFF; /* 蓝色字体 */
}

/* 输入框样式 */
QLineEdit {
    border: 1px solid #ccc; /* 边框颜色 */
    border-radius: 4px; /* 圆角 */
    padding: 5px;
    background-color: white;
    font-size: 14px;
}

/* 聚焦时 QLineEdit 样式 */
QLineEdit:focus {
    border-color: #007BFF; /* 聚焦时的边框颜色 */
    box-shadow: 0 0 5px rgba(0, 123, 255, 0.5); /* 聚焦时的阴影效果 */
}

/* 按钮样式 */
QPushButton {
    background-color: #007BFF; /* 蓝色背景 */
    color: white;
    border: none;
    border-radius: 5px; /* 圆角 */
    padding: 10px 20px; /* 按钮内边距 */
    font-size: 12px;
    cursor: pointer; /* 鼠标悬停时变成手型 */
}

/* 按钮悬停时的样式 */
QPushButton:hover {
    background-color: #0056b3; /* 悬停时的背景颜色 */
}

/* 保存按钮的特定样式 */
QPushButton#pushButton {
    background-color: #28a745; /* 绿色背景 */
}

QPushButton#pushButton:hover {
    background-color: #218838; /* 悬停时的绿色背景 */
}

/* 取消按钮的特定样式 */
QPushButton#pushButton_2 {
    background-color: #f44336; /* 红色背景 */
}

QPushButton#pushButton_2:hover {
    background-color: #d32f2f; /* 悬停时的红色背景 */
}

/* 布局间距 */
QGridLayout {
    margin: 20px;
    spacing: 10px; /* 控件之间的间距 */
}

/* 输入框和标签之间的间距 */
QGridLayout QLabel, QGridLayout QLineEdit {
    margin-bottom: 10px;
}

/* 添加间隔的样式 */
QSpacerItem {
    background-color: transparent;
}
</string>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="styleSheet">
      <string notr="true">/* 整体窗口背景颜色 */
QWidget#DoctorEditView {
    background-color: #ffffff; /* 白色背景 */
    font-family: &quot;Microsoft YaHei UI&quot;, sans-serif; /* 使用微软雅黑字体 */
    font-size: 12pt;
    border-radius: 5px; /* 圆角 */
    padding: 20px; /* 内边距 */
}

/* 标签样式 */
QLabel {
    font-size: 12pt;
    font-weight: bold;
    color: #007BFF; /* 蓝色字体 */
}

/* 输入框样式 */
QLineEdit {
    border: 1px solid #ccc; /* 边框颜色 */
    border-radius: 4px; /* 圆角 */
    padding: 5px;
    background-color: white;
    font-size: 14px;
}

/* 聚焦时 QLineEdit 样式 */
QLineEdit:focus {
    border-color: #007BFF; /* 聚焦时的边框颜色 */
    box-shadow: 0 0 5px rgba(0, 123, 255, 0.5); /* 聚焦时的阴影效果 */
}

/* 按钮样式 */
QPushButton {
    background-color: #007BFF; /* 蓝色背景 */
    color: white;
    border: none;
    border-radius: 5px; /* 圆角 */
    padding: 10px 20px; /* 按钮内边距 */
    font-size: 12px;
    cursor: pointer; /* 鼠标悬停时变成手型 */
}

/* 按钮悬停时的样式 */
QPushButton:hover {
    background-color: #0056b3; /* 悬停时的背景颜色 */
}

/* 保存按钮的特定样式 */
QPushButton#pushButton {
    background-color: #28a745; /* 绿色背景 */
}

QPushButton#pushButton:hover {
    background-color: #218838; /* 悬停时的绿色背景 */
}

/* 取消按钮的特定样式 */
QPushButton#pushButton_2 {
    background-color: #f44336; /* 红色背景 */
}

QPushButton#pushButton_2:hover {
    background-color: #d32f2f; /* 悬停时的红色背景 */
}

/* 布局间距 */
QGridLayout {
    margin: 20px;
    spacing: 10px; /* 控件之间的间距 */
}

/* 输入框和标签之间的间距 */
QGridLayout QLabel, QGridLayout QLineEdit {
    margin-bottom: 10px;
}

/* 添加间隔的样式 */
QSpacerItem {
    background-color: transparent;
}
</string>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>156</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">/* 整体窗口背景颜色 */
QWidget#DoctorEditView {
    background-color: #ffffff; /* 白色背景 */
    font-family: &quot;Microsoft YaHei UI&quot;, sans-serif; /* 使用微软雅黑字体 */
    font-size: 12pt;
    border-radius: 5px; /* 圆角 */
    padding: 20px; /* 内边距 */
}

/* 标签样式 */
QLabel {
    font-size: 12pt;
    font-weight: bold;
    color: #007BFF; /* 蓝色字体 */
}

/* 输入框样式 */
QLineEdit {
    border: 1px solid #ccc; /* 边框颜色 */
    border-radius: 4px; /* 圆角 */
    padding: 5px;
    background-color: white;
    font-size: 14px;
}

/* 聚焦时 QLineEdit 样式 */
QLineEdit:focus {
    border-color: #007BFF; /* 聚焦时的边框颜色 */
    box-shadow: 0 0 5px rgba(0, 123, 255, 0.5); /* 聚焦时的阴影效果 */
}

/* 按钮样式 */
QPushButton {
    background-color: #007BFF; /* 蓝色背景 */
    color: white;
    border: none;
    border-radius: 5px; /* 圆角 */
    padding: 10px 20px; /* 按钮内边距 */
    font-size: 12px;
    cursor: pointer; /* 鼠标悬停时变成手型 */
}

/* 按钮悬停时的样式 */
QPushButton:hover {
    background-color: #0056b3; /* 悬停时的背景颜色 */
}

/* 保存按钮的特定样式 */
QPushButton#pushButton {
    background-color: #28a745; /* 绿色背景 */
}

QPushButton#pushButton:hover {
    background-color: #218838; /* 悬停时的绿色背景 */
}

/* 取消按钮的特定样式 */
QPushButton#pushButton_2 {
    background-color: #f44336; /* 红色背景 */
}

QPushButton#pushButton_2:hover {
    background-color: #d32f2f; /* 悬停时的红色背景 */
}

/* 布局间距 */
QGridLayout {
    margin: 20px;
    spacing: 10px; /* 控件之间的间距 */
}

/* 输入框和标签之间的间距 */
QGridLayout QLabel, QGridLayout QLineEdit {
    margin-bottom: 10px;
}

/* 添加间隔的样式 */
QSpacerItem {
    background-color: transparent;
}
</string>
       </property>
       <property name="text">
        <string>ID:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="dbEditID">
       <property name="styleSheet">
        <string notr="true">/* 整体窗口背景颜色 */
QWidget#DoctorEditView {
    background-color: #ffffff; /* 白色背景 */
    font-family: &quot;Microsoft YaHei UI&quot;, sans-serif; /* 使用微软雅黑字体 */
    font-size: 12pt;
    border-radius: 5px; /* 圆角 */
    padding: 20px; /* 内边距 */
}

/* 标签样式 */
QLabel {
    font-size: 12pt;
    font-weight: bold;
    color: #007BFF; /* 蓝色字体 */
}

/* 输入框样式 */
QLineEdit {
    border: 1px solid #ccc; /* 边框颜色 */
    border-radius: 4px; /* 圆角 */
    padding: 5px;
    background-color: white;
    font-size: 14px;
}

/* 聚焦时 QLineEdit 样式 */
QLineEdit:focus {
    border-color: #007BFF; /* 聚焦时的边框颜色 */
    box-shadow: 0 0 5px rgba(0, 123, 255, 0.5); /* 聚焦时的阴影效果 */
}

/* 按钮样式 */
QPushButton {
    background-color: #007BFF; /* 蓝色背景 */
    color: white;
    border: none;
    border-radius: 5px; /* 圆角 */
    padding: 10px 20px; /* 按钮内边距 */
    font-size: 12px;
    cursor: pointer; /* 鼠标悬停时变成手型 */
}

/* 按钮悬停时的样式 */
QPushButton:hover {
    background-color: #0056b3; /* 悬停时的背景颜色 */
}

/* 保存按钮的特定样式 */
QPushButton#pushButton {
    background-color: #28a745; /* 绿色背景 */
}

QPushButton#pushButton:hover {
    background-color: #218838; /* 悬停时的绿色背景 */
}

/* 取消按钮的特定样式 */
QPushButton#pushButton_2 {
    background-color: #f44336; /* 红色背景 */
}

QPushButton#pushButton_2:hover {
    background-color: #d32f2f; /* 悬停时的红色背景 */
}

/* 布局间距 */
QGridLayout {
    margin: 20px;
    spacing: 10px; /* 控件之间的间距 */
}

/* 输入框和标签之间的间距 */
QGridLayout QLabel, QGridLayout QLineEdit {
    margin-bottom: 10px;
}

/* 添加间隔的样式 */
QSpacerItem {
    background-color: transparent;
}
</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="dbEditEmployeeNo">
       <property name="styleSheet">
        <string notr="true">/* 整体窗口背景颜色 */
QWidget#DoctorEditView {
    background-color: #ffffff; /* 白色背景 */
    font-family: &quot;Microsoft YaHei UI&quot;, sans-serif; /* 使用微软雅黑字体 */
    font-size: 12pt;
    border-radius: 5px; /* 圆角 */
    padding: 20px; /* 内边距 */
}

/* 标签样式 */
QLabel {
    font-size: 12pt;
    font-weight: bold;
    color: #007BFF; /* 蓝色字体 */
}

/* 输入框样式 */
QLineEdit {
    border: 1px solid #ccc; /* 边框颜色 */
    border-radius: 4px; /* 圆角 */
    padding: 5px;
    background-color: white;
    font-size: 14px;
}

/* 聚焦时 QLineEdit 样式 */
QLineEdit:focus {
    border-color: #007BFF; /* 聚焦时的边框颜色 */
    box-shadow: 0 0 5px rgba(0, 123, 255, 0.5); /* 聚焦时的阴影效果 */
}

/* 按钮样式 */
QPushButton {
    background-color: #007BFF; /* 蓝色背景 */
    color: white;
    border: none;
    border-radius: 5px; /* 圆角 */
    padding: 10px 20px; /* 按钮内边距 */
    font-size: 12px;
    cursor: pointer; /* 鼠标悬停时变成手型 */
}

/* 按钮悬停时的样式 */
QPushButton:hover {
    background-color: #0056b3; /* 悬停时的背景颜色 */
}

/* 保存按钮的特定样式 */
QPushButton#pushButton {
    background-color: #28a745; /* 绿色背景 */
}

QPushButton#pushButton:hover {
    background-color: #218838; /* 悬停时的绿色背景 */
}

/* 取消按钮的特定样式 */
QPushButton#pushButton_2 {
    background-color: #f44336; /* 红色背景 */
}

QPushButton#pushButton_2:hover {
    background-color: #d32f2f; /* 悬停时的红色背景 */
}

/* 布局间距 */
QGridLayout {
    margin: 20px;
    spacing: 10px; /* 控件之间的间距 */
}

/* 输入框和标签之间的间距 */
QGridLayout QLabel, QGridLayout QLineEdit {
    margin-bottom: 10px;
}

/* 添加间隔的样式 */
QSpacerItem {
    background-color: transparent;
}
</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">/* 整体窗口背景颜色 */
QWidget#DoctorEditView {
    background-color: #ffffff; /* 白色背景 */
    font-family: &quot;Microsoft YaHei UI&quot;, sans-serif; /* 使用微软雅黑字体 */
    font-size: 12pt;
    border-radius: 5px; /* 圆角 */
    padding: 20px; /* 内边距 */
}

/* 标签样式 */
QLabel {
    font-size: 12pt;
    font-weight: bold;
    color: #007BFF; /* 蓝色字体 */
}

/* 输入框样式 */
QLineEdit {
    border: 1px solid #ccc; /* 边框颜色 */
    border-radius: 4px; /* 圆角 */
    padding: 5px;
    background-color: white;
    font-size: 14px;
}

/* 聚焦时 QLineEdit 样式 */
QLineEdit:focus {
    border-color: #007BFF; /* 聚焦时的边框颜色 */
    box-shadow: 0 0 5px rgba(0, 123, 255, 0.5); /* 聚焦时的阴影效果 */
}

/* 按钮样式 */
QPushButton {
    background-color: #007BFF; /* 蓝色背景 */
    color: white;
    border: none;
    border-radius: 5px; /* 圆角 */
    padding: 10px 20px; /* 按钮内边距 */
    font-size: 12px;
    cursor: pointer; /* 鼠标悬停时变成手型 */
}

/* 按钮悬停时的样式 */
QPushButton:hover {
    background-color: #0056b3; /* 悬停时的背景颜色 */
}

/* 保存按钮的特定样式 */
QPushButton#pushButton {
    background-color: #28a745; /* 绿色背景 */
}

QPushButton#pushButton:hover {
    background-color: #218838; /* 悬停时的绿色背景 */
}

/* 取消按钮的特定样式 */
QPushButton#pushButton_2 {
    background-color: #f44336; /* 红色背景 */
}

QPushButton#pushButton_2:hover {
    background-color: #d32f2f; /* 悬停时的红色背景 */
}

/* 布局间距 */
QGridLayout {
    margin: 20px;
    spacing: 10px; /* 控件之间的间距 */
}

/* 输入框和标签之间的间距 */
QGridLayout QLabel, QGridLayout QLineEdit {
    margin-bottom: 10px;
}

/* 添加间隔的样式 */
QSpacerItem {
    background-color: transparent;
}
</string>
       </property>
       <property name="text">
        <string>编号：</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">/* 整体窗口背景颜色 */
QWidget#DoctorEditView {
    background-color: #ffffff; /* 白色背景 */
    font-family: &quot;Microsoft YaHei UI&quot;, sans-serif; /* 使用微软雅黑字体 */
    font-size: 12pt;
    border-radius: 5px; /* 圆角 */
    padding: 20px; /* 内边距 */
}

/* 标签样式 */
QLabel {
    font-size: 12pt;
    font-weight: bold;
    color: #007BFF; /* 蓝色字体 */
}

/* 输入框样式 */
QLineEdit {
    border: 1px solid #ccc; /* 边框颜色 */
    border-radius: 4px; /* 圆角 */
    padding: 5px;
    background-color: white;
    font-size: 14px;
}

/* 聚焦时 QLineEdit 样式 */
QLineEdit:focus {
    border-color: #007BFF; /* 聚焦时的边框颜色 */
    box-shadow: 0 0 5px rgba(0, 123, 255, 0.5); /* 聚焦时的阴影效果 */
}

/* 按钮样式 */
QPushButton {
    background-color: #007BFF; /* 蓝色背景 */
    color: white;
    border: none;
    border-radius: 5px; /* 圆角 */
    padding: 10px 20px; /* 按钮内边距 */
    font-size: 12px;
    cursor: pointer; /* 鼠标悬停时变成手型 */
}

/* 按钮悬停时的样式 */
QPushButton:hover {
    background-color: #0056b3; /* 悬停时的背景颜色 */
}

/* 保存按钮的特定样式 */
QPushButton#pushButton {
    background-color: #28a745; /* 绿色背景 */
}

QPushButton#pushButton:hover {
    background-color: #218838; /* 悬停时的绿色背景 */
}

/* 取消按钮的特定样式 */
QPushButton#pushButton_2 {
    background-color: #f44336; /* 红色背景 */
}

QPushButton#pushButton_2:hover {
    background-color: #d32f2f; /* 悬停时的红色背景 */
}

/* 布局间距 */
QGridLayout {
    margin: 20px;
    spacing: 10px; /* 控件之间的间距 */
}

/* 输入框和标签之间的间距 */
QGridLayout QLabel, QGridLayout QLineEdit {
    margin-bottom: 10px;
}

/* 添加间隔的样式 */
QSpacerItem {
    background-color: transparent;
}
</string>
       </property>
       <property name="text">
        <string>部门ID:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="dbEditDepartmentID">
       <property name="styleSheet">
        <string notr="true">/* 整体窗口背景颜色 */
QWidget#DoctorEditView {
    background-color: #ffffff; /* 白色背景 */
    font-family: &quot;Microsoft YaHei UI&quot;, sans-serif; /* 使用微软雅黑字体 */
    font-size: 12pt;
    border-radius: 5px; /* 圆角 */
    padding: 20px; /* 内边距 */
}

/* 标签样式 */
QLabel {
    font-size: 12pt;
    font-weight: bold;
    color: #007BFF; /* 蓝色字体 */
}

/* 输入框样式 */
QLineEdit {
    border: 1px solid #ccc; /* 边框颜色 */
    border-radius: 4px; /* 圆角 */
    padding: 5px;
    background-color: white;
    font-size: 14px;
}

/* 聚焦时 QLineEdit 样式 */
QLineEdit:focus {
    border-color: #007BFF; /* 聚焦时的边框颜色 */
    box-shadow: 0 0 5px rgba(0, 123, 255, 0.5); /* 聚焦时的阴影效果 */
}

/* 按钮样式 */
QPushButton {
    background-color: #007BFF; /* 蓝色背景 */
    color: white;
    border: none;
    border-radius: 5px; /* 圆角 */
    padding: 10px 20px; /* 按钮内边距 */
    font-size: 12px;
    cursor: pointer; /* 鼠标悬停时变成手型 */
}

/* 按钮悬停时的样式 */
QPushButton:hover {
    background-color: #0056b3; /* 悬停时的背景颜色 */
}

/* 保存按钮的特定样式 */
QPushButton#pushButton {
    background-color: #28a745; /* 绿色背景 */
}

QPushButton#pushButton:hover {
    background-color: #218838; /* 悬停时的绿色背景 */
}

/* 取消按钮的特定样式 */
QPushButton#pushButton_2 {
    background-color: #f44336; /* 红色背景 */
}

QPushButton#pushButton_2:hover {
    background-color: #d32f2f; /* 悬停时的红色背景 */
}

/* 布局间距 */
QGridLayout {
    margin: 20px;
    spacing: 10px; /* 控件之间的间距 */
}

/* 输入框和标签之间的间距 */
QGridLayout QLabel, QGridLayout QLineEdit {
    margin-bottom: 10px;
}

/* 添加间隔的样式 */
QSpacerItem {
    background-color: transparent;
}
</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">/* 整体窗口背景颜色 */
QWidget#DoctorEditView {
    background-color: #ffffff; /* 白色背景 */
    font-family: &quot;Microsoft YaHei UI&quot;, sans-serif; /* 使用微软雅黑字体 */
    font-size: 12pt;
    border-radius: 5px; /* 圆角 */
    padding: 20px; /* 内边距 */
}

/* 标签样式 */
QLabel {
    font-size: 12pt;
    font-weight: bold;
    color: #007BFF; /* 蓝色字体 */
}

/* 输入框样式 */
QLineEdit {
    border: 1px solid #ccc; /* 边框颜色 */
    border-radius: 4px; /* 圆角 */
    padding: 5px;
    background-color: white;
    font-size: 14px;
}

/* 聚焦时 QLineEdit 样式 */
QLineEdit:focus {
    border-color: #007BFF; /* 聚焦时的边框颜色 */
    box-shadow: 0 0 5px rgba(0, 123, 255, 0.5); /* 聚焦时的阴影效果 */
}

/* 按钮样式 */
QPushButton {
    background-color: #007BFF; /* 蓝色背景 */
    color: white;
    border: none;
    border-radius: 5px; /* 圆角 */
    padding: 10px 20px; /* 按钮内边距 */
    font-size: 12px;
    cursor: pointer; /* 鼠标悬停时变成手型 */
}

/* 按钮悬停时的样式 */
QPushButton:hover {
    background-color: #0056b3; /* 悬停时的背景颜色 */
}

/* 保存按钮的特定样式 */
QPushButton#pushButton {
    background-color: #28a745; /* 绿色背景 */
}

QPushButton#pushButton:hover {
    background-color: #218838; /* 悬停时的绿色背景 */
}

/* 取消按钮的特定样式 */
QPushButton#pushButton_2 {
    background-color: #f44336; /* 红色背景 */
}

QPushButton#pushButton_2:hover {
    background-color: #d32f2f; /* 悬停时的红色背景 */
}

/* 布局间距 */
QGridLayout {
    margin: 20px;
    spacing: 10px; /* 控件之间的间距 */
}

/* 输入框和标签之间的间距 */
QGridLayout QLabel, QGridLayout QLineEdit {
    margin-bottom: 10px;
}

/* 添加间隔的样式 */
QSpacerItem {
    background-color: transparent;
}
</string>
       </property>
       <property name="text">
        <string>姓名：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="dbEditName">
       <property name="styleSheet">
        <string notr="true">/* 整体窗口背景颜色 */
QWidget#DoctorEditView {
    background-color: #ffffff; /* 白色背景 */
    font-family: &quot;Microsoft YaHei UI&quot;, sans-serif; /* 使用微软雅黑字体 */
    font-size: 12pt;
    border-radius: 5px; /* 圆角 */
    padding: 20px; /* 内边距 */
}

/* 标签样式 */
QLabel {
    font-size: 12pt;
    font-weight: bold;
    color: #007BFF; /* 蓝色字体 */
}

/* 输入框样式 */
QLineEdit {
    border: 1px solid #ccc; /* 边框颜色 */
    border-radius: 4px; /* 圆角 */
    padding: 5px;
    background-color: white;
    font-size: 14px;
}

/* 聚焦时 QLineEdit 样式 */
QLineEdit:focus {
    border-color: #007BFF; /* 聚焦时的边框颜色 */
    box-shadow: 0 0 5px rgba(0, 123, 255, 0.5); /* 聚焦时的阴影效果 */
}

/* 按钮样式 */
QPushButton {
    background-color: #007BFF; /* 蓝色背景 */
    color: white;
    border: none;
    border-radius: 5px; /* 圆角 */
    padding: 10px 20px; /* 按钮内边距 */
    font-size: 12px;
    cursor: pointer; /* 鼠标悬停时变成手型 */
}

/* 按钮悬停时的样式 */
QPushButton:hover {
    background-color: #0056b3; /* 悬停时的背景颜色 */
}

/* 保存按钮的特定样式 */
QPushButton#pushButton {
    background-color: #28a745; /* 绿色背景 */
}

QPushButton#pushButton:hover {
    background-color: #218838; /* 悬停时的绿色背景 */
}

/* 取消按钮的特定样式 */
QPushButton#pushButton_2 {
    background-color: #f44336; /* 红色背景 */
}

QPushButton#pushButton_2:hover {
    background-color: #d32f2f; /* 悬停时的红色背景 */
}

/* 布局间距 */
QGridLayout {
    margin: 20px;
    spacing: 10px; /* 控件之间的间距 */
}

/* 输入框和标签之间的间距 */
QGridLayout QLabel, QGridLayout QLineEdit {
    margin-bottom: 10px;
}

/* 添加间隔的样式 */
QSpacerItem {
    background-color: transparent;
}
</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
